sycl/plugins/level_zero/pi_level_zero.cpp

```diff
@@ -2239,6 +2239,12 @@ pi_result piDeviceRelease(pi_device Device) {
 pi_result piDeviceGetInfo(pi_device Device, pi_device_info ParamName,
                           size_t ParamValueSize, void *ParamValue,
                           size_t *ParamValueSizeRet) {
+  // TODO: This is a work-around for missing UR info query. Replace with proper
+  //       UR mapping when available.
+  if (ParamName == PI_EXT_INTEL_DEVICE_INFO_MEM_CHANNEL_SUPPORT) {
+    ReturnHelper ReturnValue(ParamValueSize, ParamValue, ParamValueSizeRet);
+    return ReturnValue(pi_bool{false});
+  }
```

Comment:
@smaslov-intel - I did a similar work-around in https://github.com/intel/llvm/pull/8954. Is there a better way of doing it?

Reply:
@steffenlarsen : Yes, let's not add new stuff here.
Please temporary add UR extension here:
https://github.com/intel/llvm/blob/f81499ecad464bd03d7f50e95004eeed1e85dbc9/sycl/plugins/unified_runtime/ur/ur.hpp#L23

And note @kbenzie to promote it into UR.
The the handling would naturally go into https://github.com/intel/llvm/blob/sycl/sycl/plugins/unified_runtime/ur/adapters/level_zero/ur_level_zero.cpp

Reply:
I was not aware of this list of constants until now. Do lines [23-57](https://github.com/intel/llvm/blob/f81499ecad464bd03d7f50e95004eeed1e85dbc9/sycl/plugins/unified_runtime/ur/ur.hpp#L23-L57) encapsulate these `UR_EXT_*` symbols are are there additional ones?

Reply:
Found https://github.com/oneapi-src/unified-runtime/issues/374 which seems to cover theses, added mention of this.